src/modules/dispatcher/dispatcher.c

```diff
@@ -1798,15 +1798,19 @@ static const char *dispatcher_rpc_add_doc[2] = {
  */
 static void dispatcher_rpc_add(rpc_t *rpc, void *ctx)
 {
-	int group, flags;
+	int group, flags, nparams;
 	str dest;
-  str attrs;
+	str attrs;
 
 	flags = 0;
 
-	if(rpc->scan(ctx, "dS*d", &group, &dest, &flags, &attrs) < 3) {
+	nparams = rpc->scan(ctx, "dS*dS", &group, &dest, &flags, &attrs);
+	if(nparams < 2) {
 		rpc->fault(ctx, 500, "Invalid Parameters");
 		return;
+	} else if (nparams < 4) {
```

Comment:
I think the condition here has to be `(nparams < 4)`.

Reply:
right, fixed and force pushed.